README.md

```diff
@@ -201,6 +201,27 @@ This section has been eliminated in the Meteor version, because it does not spec
     };
     ```
 
+  - [3.8](#3.8) <a name='3.8'></a> Use object spreads `...` to copy objects.
```

Comment:
You would avoid a useful feature because an Emacs mode that some people use hasn't bothered supporting it yet? Pretty sure we can fix that, in any number of ways. I use https://github.com/benjamn/dot-files/blob/master/elisp/js.el and I don't have any problems with object spreads (though it is less aggressive about actually parsing your code than js2-mode).